libs/shared/src/lib/components/header/header.tsx

```diff
@@ -1,37 +1,72 @@
+import { Link, useLocation } from 'react-router-dom';
+import { useState } from 'react';
 
-export function Header(): JSX.Element {
+export function Header(props: { img: string }): JSX.Element {
+  const currentLocation = useLocation().pathname;
+  const [isActive, setActive] = useState(currentLocation);
   return (
     <Grid container item>
-      <Grid item container justifyContent='center' xs={12} bgcolor={theme('light').palette.black.dark} py='6px'>
-       <Typography />
+      <Grid item container xs={12} bgcolor={theme('light').palette.black.dark} py='6px'>
+       <Grid item container direction='row' gap='12px' justifyContent='center'>
+         <Typography className={styles['blinking']} label='Free shipping' variant='body1'/>
+         <Typography label='order from all item' variant='body1' color={theme('light').palette.white.main} />
+       </Grid>
       </Grid>
-      <Grid item container direction='column' px='48px' py='28px'>
-
-        <Grid item container direction='row' px='28px'>
+      <Grid item container direction='column' px='48px' py='12px'>
+        <Grid item container direction='row' px='28px' pb='12px'>
           <Grid item container xs={4}>
-            Logo
+            <Box width='90px' height='60px'>
+              <img src={props?.img} className='object-fit' />
+            </Box>
           </Grid>
-          <Grid item container  xs={4} justifyContent='center' >
-            <Input />
+          <Grid item container  xs={4} justifyContent='center' alignItems='center'>
+            <Input placeholder='Search our store'
+                   variant='outlined'
+                   className='width--full custom-search-input'
+                   InputProps={ {
+                     startAdornment: (
+                       <InputAdornment position='end'>
+                         <SearchOutlinedIcon />
+                       </InputAdornment>
+                     ),
+                   } }
+            />
           </Grid>
-          <Grid item container direction='row' xs={4} justifyContent='end' gap='12px'>
-            <Box>
-              cart
-            </Box>
-            <Box>
-              Fav
-            </Box>
-            <Box>
-              Acc
+          <Grid item container direction='row' xs={4} alignItems='center' justifyContent='end' gap='18px'>
+              <Badge badgeContent={4} color='primary' className='cursor--pointer' >
+                <ShoppingCartOutlinedIcon fontSize='medium' />
+              </Badge>
+              <Badge badgeContent={4}  color='primary' className='cursor--pointer'>
+                <FavoriteBorderOutlinedIcon fontSize='medium' />
+              </Badge>
+            <Box pl='22px'>
+              <AccountCircleOutlinedIcon fontSize='large' />
             </Box>
           </Grid>
         </Grid>
-        <Grid item container px='28px'>
-          hi
+        <Box px='28px'>
+          <Divider />
```

Comment:
why can't you use px='28px' in Divider component.

Reply:
divider ga sho mala 
Should i make custom divider?